respec-config.js

```diff
@@ -30,7 +30,7 @@ var respecConfig = {
   // previousDiffURI: "",
 
   // If there a publicly available Editors Draft, this is the link
-  edDraftURI: 'https://w3c.github.io/aria-practices/',
+  edDraftURI: 'https://w3c.github.io/aria-practices-1.2/',
```

Comment:
This won't work ... the path in the gh-pages URI is the repo name. We don't have a separate repo for 1.2. The only option here is to use rawgit. Or, another possibility is to restructure the entire gh-pages branch to have a 1.1 dir and a 1.2 dir and then reconfig travis so that it puts stuff in the right dir ... hmm, but then that would mean restructuring master as well. We would have to separate dirs in master. For now, the simplest, I guess, is to use rawgit, which is not the greatest idea b/c github says not to drive public traffic to rawgit.